tests/manager/manager-executeWriteCommand_error-002.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+MongoDB\Driver\Manager::executeWriteCommand() throws CommandException for invalid writeConcern
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('REPLICASET'); CLEANUP(REPLICASET); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(REPLICASET);
+
+$document = ['foo' => ['bar']];
+
+$command = new MongoDB\Driver\Command([
+    'findAndModify' => COLLECTION_NAME,
+    'query' => ['_id' => 'foo'],
+    'update' => $document,
+    'upsert' => true,
+    'new' => true,
+]);
+
+try {
+    $manager->executeWriteCommand(DATABASE_NAME, $command, ['writeConcern' => new MongoDB\Driver\WriteConcern("minority")]);
+} catch (MongoDB\Driver\Exception\CommandException $e) {
+	printf("CommandException: %s\n", $e->getMessage());
```

Comment:
I'd suggest adding the exception code to this output, so we can test that `phongo_execute_command()` successfully sets that when throwing the exception.

I typically use `printf("%s(%d): %s\n", get_class($e), $e->getCode(), $e->getMessage())` in my own scripts, but you can certainly hard-code the short class name here.